test/e2e/admin_portal.go

```diff
@@ -0,0 +1,205 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"                           //nolint
+	. "github.com/onsi/gomega"                           //nolint
+	. "github.com/serge1peshcoff/selenium-go-conditions" //nolint
+	. "github.com/tebeka/selenium"                       //nolint
+)
+
+var _ = Describe("Admin Portal E2E Testing", func() {
+	BeforeEach(skipIfNotInDevelopmentEnv)
+	var wdPoint *WebDriver
+	var wd WebDriver
+	var host string
+	JustBeforeEach(func() {
+		host, wdPoint = adminPortalSessionSetup()
+		wd = *wdPoint
+		wd.Get(host + "/v2")
+		wd.Refresh()
+	})
+
+	AfterEach(func() {
+		if wd != nil {
+			wd.Quit()
+		}
+	})
+	It("Should be able to populate cluster data correctly", func() {
+		wd.Wait(ElementIsLocated(ByCSSSelector, "div[data-automation-key='name']"))
+
+		cluster, err := wd.FindElement(ByCSSSelector, "div[data-automation-key='name']")
+		if err != nil {
+			panic(err)
+		}
+
+		Expect(cluster.Text()).To(Equal(os.Getenv("CLUSTER")))
+	})
+
+	It("Should be able to filter cluster data correctly", func() {
+		wd.Wait(ElementIsLocated(ByCSSSelector, "div[data-automation-key='name']"))
+
+		filter, err := wd.FindElement(ByCSSSelector, "input[placeholder='Filter on resource ID']")
+		if err != nil {
+			panic(err)
+		}
+
+		// Set filter so it doesn't match cluster name
+		filter.SendKeys("Incorrect Cluster")
+
+		wd.Wait(ElementIsLocated(ByCSSSelector, "span.itemsCount-162"))
+		text, err := wd.FindElement(ByCSSSelector, "span.itemsCount-162")
+		if err != nil {
+			panic(err)
+		}
+
+		Expect(text.Text()).To(Equal("Showing 0 items"))
+	})
+
+	It("Should be able to populate cluster info panel correctly", func() {
+		testValues := [17]string{
```

Comment:
where do these test values come from?

Reply:
Ooh, these are just the random hard coded values from my local environment. I definitely missed going back to fix them because the test seems to pass on the CT... Definitely something not quite right there...

Reply:
I have modified these tests, the css selector was wrong and returning nothing, so these should work now